ci/azure-steps-windows.yml

```
@@ -0,0 +1,24 @@
+steps:
+- script: |
+    echo "##vso[task.prependpath]$CONDA/bin"
+  displayName: Prepare conda
+
+- script: |
+    conda env update -n uarray -f .conda/environment$(python_version).yml
+  displayName: Create environment.
+
+- script: |
+    source activate uarray
```

With recent `conda` this should be:
```suggestion
    conda activate uarray
```
On Windows I'd recommend using the latest 4.7.6 as there have been various issues with earlier versions in the 4.7* series.